--- a/xbmc/input/XBMC_keysym.h
+++ b/xbmc/input/XBMC_keysym.h
@@ -114,7 +114,7 @@ typedef enum {
   XBMCK_PIPE        = 0x7C,
   XBMCK_RIGHTBRACE  = 0x7D,
   XBMCK_TILDE       = 0x7E,
-  XBMCK_DELETE = 0x7F,
+  XBMCK_DELETE      = 0x7F,
   // End of ASCII mapped keysyms

   // Multimedia keys
@@ -242,7 +242,9 @@ typedef enum {
   XBMCK_PLAY              = 341,
   XBMCK_SHUFFLE           = 342,
   XBMCK_FASTFORWARD       = 343,
-
+  XBMCK_FASTREVERSE       = 344,
+  XBMCK_SLOWFORWARD       = 345,
+  XBMCK_SLOWREVERSE       = 346,
   XBMCK_LAST
 } XBMCKey;

